mlrun/launcher/client.py

```diff
@@ -47,7 +47,9 @@ def prepare_image_for_deploy(runtime: "mlrun.runtimes.BaseRuntime"):
         If build is needed, set the image as the base_image for the build.
         If image is not given set the default one.
         """
-        if runtime.kind in mlrun.runtimes.RuntimeKinds.nuclio_runtimes():
+        if runtime.kind in set(
+            mlrun.runtimes.RuntimeKinds.nuclio_runtimes()
+        ).difference(set(mlrun.runtimes.RuntimeKinds.application)):
```

Comment:
Even further on this - can you give this reduced set a meaningful designation? For example "pure nuclio runtimes" or something? If so, I would put this in another accessor in `RuntimeKinds` rather than calculate it here.

Reply:
👍